lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -371,88 +397,82 @@ CRI_TICF_BEFORE_SUCCESS:
   response:
     type: process
     lambda: call-ticf-cri
+  parent: CRI_TICF_STATE
   events:
     next:
       targetState: IPV_SUCCESS_PAGE
-    enhanced-verification:
-      targetState: PYI_NO_MATCH
-    fail-with-ci:
-      targetState: PYI_NO_MATCH
-    error:
-      targetState: ERROR
 
 CRI_TICF_BEFORE_REUSE:
   response:
     type: process
     lambda: call-ticf-cri
+  parent: CRI_TICF_STATE
   events:
     next:
       targetState: IPV_IDENTITY_REUSE_PAGE
       checkFeatureFlag:
         deleteDetailsEnabled:
           targetState: IPV_IDENTITY_REUSE_PAGE_TEST
-    enhanced-verification:
-      targetState: PYI_NO_MATCH
-    fail-with-ci:
-      targetState: PYI_NO_MATCH
-    error:
-      targetState: ERROR
+
+CRI_TICF_BEFORE_OPERATIONAL_PROFILE_REUSE:
+  response:
+    type: process
+    lambda: call-ticf-cri
```

Comment:
I briefly tried making `response` part of the parent as well, but it needs a bit of fiddling to get it to work (including in the journey map). Maybe something to look at after the journey split